tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/UpToDate/BuildUpToDateCheckTests.cs

```diff
@@ -1031,12 +1038,18 @@ public async Task IsUpToDateAsync_False_CopyToOutputDirectorySourceIsNewerThanDe
         [Fact]
         public async Task IsUpToDateAsync_False_CopyToOutputDirectorySourceDoesNotExist()
         {
-            var sourceSnapshot = new Dictionary<string, IProjectRuleSnapshotModel>
+            var projectSnapshot = new Dictionary<string, IProjectRuleSnapshotModel>
             {
-                [Content.SchemaName] = ItemWithMetadata("Item1", "CopyToOutputDirectory", "PreserveNewest")
+                [CopyToOutputDirectoryItem.SchemaName] = new IProjectRuleSnapshotModel
+                {
+                    Items = ImmutableStringDictionary<IImmutableDictionary<string, string>>.EmptyOrdinal
+                        .Add("C:\\Dev\\Solution\\Project\\Item1", ImmutableStringDictionary<string>.EmptyOrdinal
```

Comment:
>C [](start = 30, length = 1)

Do we always have access to c:? i was just thinking that in CI we might not.

Reply:
In this implementation we check CopyToOutputDirectory items from jointRuleUpdate (projectSnapshot) which needs to receive a[ rooted path](https://github.com/dotnet/project-system/blob/482f68977fd3ade7270cb3b8995565b926d05b73/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.cs#L522-L529). 

In the unit test this rooted path might represent a file from  project reference at path "C:\\Dev\\Solution\\Project\\Item1" marked as PreserveNewest.

Reply:
These are just made up paths, they don't need to exist.